src/Form/Traits/AllowedLinkClassesTrait.php

```
@@ -43,16 +46,6 @@ public function getAllowedTypes(): array
      */
     private function validateTypes(array $types): bool
     {
-        if (empty($types)) {
-            throw new InvalidArgumentException(
```

Still need to throw this exception if developers pass in an empty array - see acceptance criteria

---

Ah yup my bad, didn't need to change any PHP in the end, have reverted